addons/event_sale/tests/test_event_sale.py

```diff
@@ -179,6 +179,7 @@ def test_event_crm_sale(self):
         # ------------------------------------------------------------
 
         ticket2_line.write({'product_uom_qty': 3})
+        ticket2_line.write({'price_unit': 50})
```

Comment:
You're fixing a specific value in the middle of a test to make it work, this will reduce the test usefulness and coverage, you should investigate more deeply the problem (and eventually change something in the test setup, but not in the flow IMHO).

Reply:
done :)